src/style.css

```diff
@@ -0,0 +1,123 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avanir";
+  font-weight: 600;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  box-sizing: border-box;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto;
+}
+
+.search-form-big {
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+}
+
+.search-form-small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-text_input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.search-text_input-big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  box-sizing: border-box;
+}
+
+.search-text_input-small {
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search-text-label {
+  position: relative;
+  display: block;
+  width: 100%;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.search-text-label-big::after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-text-label-small::after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-text-label::after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.search-text_input::placeholder {
+  font: 300px;
```

Comment:
Doesn't seem like a valid style for me.
I suppose you meant to add `font-weight`